wordle-web/components/GameBoardGuess.vue

```diff
@@ -1,18 +1,52 @@
 <template>
-  <div>
-    <LetterResult
-      v-for="(letter, k) of guess.letters"
-      :key="k"
-      :letter="letter"
-    />
-  </div>
+  <v-row dense class="justify-center">
+    <v-col :class="isShaking ? 'shaking' : ''" cols="auto" v-for="(letter, k) of guess.letters" :key="k">
+      <LetterResult :letter="letter" />
+    </v-col>
+  </v-row>
 </template>
 
 <script setup lang="ts">
 import { Word } from "~/scripts/word";
 import { defineProps } from "vue";
 
-defineProps<{
+const props = defineProps<{
   guess: Word;
 }>();
+
+const isShaking = ref(false);
+
+watch(() => props.guess.isFilled,
+() => {
+  if(!props.guess.isFilled){
+    isShaking.value = true;
+    setTimeout(() => {
+      isShaking.value = false;
+    }, 300);
+  }
+});
```

Comment:
```suggestion
watch(() => props.guess.isFilled,
(value) => {  // This function will receive the value you're watching
  if(!value){  // and should be used here
    isShaking.value = true;
    setTimeout(() => {
      isShaking.value = false;
    }, 300);
  }
});
```
Two things:
1. When implementing a watcher, it's usually preferred to use the value provided in the callback rather than directly from the reactive source. It reduces dependency on the outer scope and is a bit more explicit that `value` represents the new state at the time of the change. This callback can take an additional parameter as well (not needed in this case, but just a tidbit of info for you): (newValue, oldValue) => {}
2. This condition doesn't appear to be set up quite right, as the shake only performs when a guess is completely filled and then the last character is removed.